examples/key_from_raw_data.cpp

```diff
@@ -27,21 +27,21 @@ int main()
     // the declaration above, we have to ignore the first byte as we
     // need to set a special tag byte here for pgp to recognize it
     // note: error checks are skipped here for brevity
-    crypto_box_keypair(public_key_data.data() + 1, secret_key_data.data());
+    crypto_sign_keypair(public_key_data.data() + 1, secret_key_data.data());
 
     // libsodium puts both the secret key and the public key data in
     // the secret key buffer, which pgp does not need, therefore the
     // extra data must be truncated before creating the key
     secret_key_data.resize(32);
 
     // now set the tag byte required for pgp
-    public_key_data[0] = 0x04;
+    public_key_data[0] = 0x40;
 
     // define the creation time of this key (as a unix timestamp)
-    // and the expiration time (another unix timestamp), we define
-    // the key as being valid for 3600 seconds (an hour);
-    uint32_t creation = 1559737787;
-    uint32_t expiration = creation + 3600;
+    // and the expiration period (the amount of seconds which the key will be valid after it's creation)
+    // we definethe key as being valid for 3600 seconds (an hour);
+    uint32_t creation = 1591606967;
+    uint32_t expiration = 3600;
```

Comment:
You can set expiration to e.g. 157784630 seconds (~ 5 years) to ensure that users which try this out will actually be able to import a valid key for a very long time.

Reply:
Or even better, set the creation timestamp to the current time and we don't have to modify this in five years